docs/source/usersguide/input.rst

```diff
@@ -374,37 +374,98 @@ OpenMOC does not place a limit on the hierarchical depth - or number of nested u
+
+The following code snippet illustrates how a user may designate a positive integral number of rings and sectors for fuel pin ``Cells`` and a positive integral number of sectors with no rings for moderator ``Cells`` using the ``Cell.setNumRings(...)`` and ``Cell.setNumSectors(...)`` class methods.
 
 .. code-block:: python
 
-    # Subdivide the fuel region into 3 rings and 8 angular sectors
+    # Subdivide the fuel region into 3 rings and 12 angular sectors
     fuel.setNumRings(3)
-    fuel.setNumSectors(8)
+    fuel.setNumSectors(12)
 
-    # Subdivide the moderator region into 8 angular sectors
+    # Subdivide the moderator region into 16 angular sectors
+    moderator.setNumSectors(16)
+
+The plots shown below illustrate the pin cell material layout (left) and flat source region layout (right) where the flat source regions have been discretized using 3 equal volume rings and 12 sectors in the fuel and 16 sectors in the moderator.
+
+.. _figure_pin_cell_fsrs:
+
+.. table::
+
+   +--------------------------------------------------------+--------------------------------------------------------+
+   | .. _figa:                                              | .. _figb:                                              |
+   |                                                        |                                                        |
+   | .. image:: ../../img/pin-cell-fsrs-1.png               | .. image:: ../../img/pin-cell-fsrs-2.png               |
+   |   :width: 50 %                                         |   :width: 50 %                                         |
+   |   :align: right                                        |   :align: left                                         |
+   +--------------------------------------------------------+--------------------------------------------------------+
+
+The user may wish to capture gradients in the moderator by adding rings in the moderator. The following code snippet repeats the scenario above but with 2 rings in the moderator.
+
+.. code-block:: python
+
+    # Subdivide the fuel region into 3 rings and 12 angular sectors
+    fuel.setNumRings(3)
+    fuel.setNumSectors(12)
+
+    # Subdivide the moderator region into 2 rings 16 angular sectors
+    moderator.setNumRings(2)
     moderator.setNumSectors(16)
-   
-The pin cell materials are illustrated on the left below, while the flat source regions with 3 equal volume rings and 8 sectors in the fuel, and 16 sectors in the moderator, are displayed on the right.
 
-.. _figure_fluxes:
+Again, the pin cell materials are illustrated below on the left, while the flat source regions are displayed on the right with 2 rings now present in the moderator.
```

Comment:
ditto, I personally find the image filenames confusing since they don't seem to match up with the text